modules/repository-datastax/src/main/java/com/intuit/wasabi/repository/cassandra/impl/CassandraExperimentRepository.java

```diff
@@ -353,33 +360,18 @@ protected Experiment internalGetExperiment(Application.Name appName,
      */
     @Override
     public Experiment.ID createExperiment(NewExperiment newExperiment) {
-
-        validator.validateNewExperiment(newExperiment);
-
-        // Ensure no other experiment is using the label
```

Comment:
@bargenilesh As discussed, please add a comment in the code that `appName` and `label` have a unique constraint set in MySQL and we are depending on it to check we are not inserting duplicate entries.